docs/runner/events_list.rst

```diff
@@ -0,0 +1,25 @@
+.. _events_list_yamlref:
+
+Events List Story Runner YAML
+...........................................
+
+'asset_name' is optional and must be the name of a previously created asset.
```

Comment:
Same change as above - should use the same wording

Reply:
fixed